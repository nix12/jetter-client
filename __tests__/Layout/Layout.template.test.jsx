import React from 'react';
import { shallow } from 'enzyme';
import Template from '../../templates/Layout/Layout.template';

describe('Template', () => {
  it('should render Template component', () => {
    const props = {
      onTryAutoSignup: jest.fn()
    };

    expect(shallow(<Template {...props} />)).toMatchSnapshot();
  });
});
