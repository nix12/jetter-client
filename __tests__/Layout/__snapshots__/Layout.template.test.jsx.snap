// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Template should render Template component 1`] = `ShallowWrapper {}`;
